level1/expressions-types-values.md

```diff
@@ -100,38 +100,201 @@ Here are three general problem solving strategies:
 
 ## Literals
 
-See Creative Scala Chapter 2
+A literal is an expression that evaluates to itself.
 
+Consider:
+
+```
+2
+```
+
+```
+1 + 1
+```
+
+> Which of these is a literal expression?
+*Hopefully, you'll get the answer 2.  If not, ask the student to evaluate both expressions*
+
+
+Text is also a literal expression.
+
+```
+"To be fond of dancing"
+```
+
+> What is the type of this literal?  How did you find this out?
+
+*Hopefully, your student will say this has the type String*
+*If not, try and get them to evaluate this expression in a repl and read the type*
 
 ## Objects and Method Calls
 
-See Creative Scala Chapter 2
+The operations on an object are called it's methods.
```

Comment:
I'm not sure if it's worth talking about "objects" at all.  What do you think about saying that operations are defined by the type of expression, and leaving out the term "object" until we introduce the idea of data?

Reply:
We need to introduce objects fairly early, because they have to wrap everything in an object to get it compile. We don't have to introduce the term now, though. I'm neutral here.

Reply:
I would favour leaving it out and introducing it early in the next chapter.  This lesson seems jargon-heavy to those with no prior programming experience.